app-gateway.tf

```diff
@@ -18,6 +18,9 @@ module "appGw" {
   wafName           = "${var.product}-appGW"
   resourcegroupname = "${azurerm_resource_group.rg.name}"
   use_authentication_cert = "true"
+  firewall_mode = "Detection"
+  file_upload_limit_mb = "50"
+  max_request_body_size_kb = "128"
 
   # vNet connections
   gatewayIpConfigurations = [
```

Comment:
Ben I remember you mentioning having increased the idle time. Shouldn't we have that in code as well?

Reply:
that is upped in the branch on the module we're using